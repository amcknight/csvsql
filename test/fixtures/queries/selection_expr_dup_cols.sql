SELECT first_name ++ " B" ++ first_name ++ " " ++ last_name aS long_name, age * age * 2 AS overbig_age
FROM test/fixtures/tables/people
WHERE last_name = "McKnight"
