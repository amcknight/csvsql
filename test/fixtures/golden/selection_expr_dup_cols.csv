long_name, overbig_age
Andrew BAndrew McKnight, 2450
July BJuly McKnight, 0
